Packages/PackageSources/meridian-simcache/html_ui/InGamePanels/SimCache/SimCache.js

```diff
@@ -70,19 +61,10 @@ class SimCachePanel extends UIElement {
         }, 5000);
     }
 
-    updateCacheTitle(title) {
```

Comment:
Is this the single line function you mentioned previously and didn't like? Seeing the context here I think it's quite nice and self-documenting rather than changing the inner text directly on line 41.

Reply:
I had added an intermediate one, but I see your point!  I've added some more functions with more self-documenting names. :)